Gemfile

```diff
@@ -50,12 +50,17 @@ gem 'bootsnap', require: false
 
```

Comment:
![Capture0](https://user-images.githubusercontent.com/93039359/206868746-48846388-6a8a-4fdb-80ac-4da62de213b2.PNG)

- [ ] You have a link of your license file in your readme file 😊 but it is broken 😨 Kindly fix this link to make your readme file descriptive and license file easily accessible to all viewers 👍